public/assets/template/delete-confirm.html

```diff
@@ -1,3 +1,4 @@
 <div class="btn-toolbar">
+    <div class="text-danger confirmation-error-text" ng-if="errorMessage" lc-message="{{errorMessage}}"></div>
```

Comment:
True. Also I've removed 'btn-toolbar' class from parent container - it seems to be misuse.